### PR TITLE
QDS does not capture DDL by design

### PR DESCRIPTION
**Pull request by miobrado**
Highlight that QDS does not capture DDL **by design** but will still capture all underlying DML statements executed internally as part of DDL statement.

### CODE REVIEW

**Comment by miobrado** [comment]
@MashaMSFT for review

**Comment by Court72** [comment]
@WilliamDAssafMSFT

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/relational-databases/performance/monitoring-performance-by-using-the-query-store.md [hunk 1]
```diff
@@ -75,6 +75,8 @@ Since the Query Store retains multiple execution plans per query, it can enforce
 > [!NOTE]  
 > Query Store collects plans for DML Statements such as SELECT, INSERT, UPDATE, DELETE, MERGE, and BULK INSERT.
 >
+> By design, Query Store does not collect plans for DDL Statements such as CREATE INDEX, etc. Still, it captures cumulative resource consumption by collecting plans for the underlying DML Statements. For example, DML Statements executed internally to populate a created index.
```
Comment (WilliamDAssafMSFT) [inline docs/relational-databases/performance/monitoring-performance-by-using-the-query-store.md]:
```suggestion
> By design, Query Store does not collect plans for DDL statements such as CREATE INDEX, etc. Query Store captures cumulative resource consumption by collecting plans for the underlying DML statements. For example, Query Store may display the SELECT and INSERT statements executed internally to populate a new index.
```

**Comment by WilliamDAssafMSFT** [comment]
Updated! Thanks for the edit, @miobrado. Will sign off after the build completes.

**Comment by WilliamDAssafMSFT** [comment]
#sign-off

**Pull request merged**